### PR TITLE
MALZ 13 to 21

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Further mss

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

**Review by eu-genia — CHANGES_REQUESTED** [review]
Note there are XML errors!

https://github.com/BetaMasaheft/Manuscripts/actions/runs/8645164728/job/23701790179?pr=2371

File: AlaqmaMaryamZadibaKokweh/MALZ021.xml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                                     <material key="wood"/>
+                                     <material key="leather"/>
+                                     <material key="textile"/>
+                                 </decoNote>
+                                 
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>
+                     
+                    <additions>
```
Comment (eu-genia) [inline AlaqmaMaryamZadibaKokweh/MALZ021.xml:91]: additions go inside physDesc

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: AlaqmaMaryamZadibaKokweh/MALZ021.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply AlaqmaMaryamZadibaKokweh/MALZ021.xml:91]: ok

File: AlaqmaMaryamZadibaKokweh/MALZ020.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                       <decoDesc>
+                           <decoNote xml:id="d1" type="miniature">          
+                               <locus target="#1v"/>
+                               <desc>A crude miniature of St Michael</desc>
+                               
+                               <decoNote xml:id="d2" type="miniature">          
+                                   <locus target="#4v" cert="low"/>
+                                   <desc>A crude drawing of an equestrian saint</desc>
+                           </decoNote>
+
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply AlaqmaMaryamZadibaKokweh/MALZ020.xml:86]: ok

**Pull request merged**